src/common/components/Sidebar/Sidebar.tsx

```diff
@@ -37,6 +37,11 @@ const SIDEBAR_ITEMS = [
     icon: <DealsIcon size={16} />,
     href: "/deals",
   },
+  {
+    label: "Coming Soon",
+    icon: <DealsIcon size={16} />,
+    href: "/coming-soon",
+  },
```

Comment:
issue: this sidebar item is not needed!

Reply:
This one was not me but it was used for easy access. Will remove it